dd-java-agent/instrumentation/elasticsearch/transport-5.3/src/main/java/datadog/trace/instrumentation/elasticsearch5_3/TransportActionListener.java

```diff
@@ -34,7 +34,7 @@ private void onRequest(final ActionRequest request) {
     if (request instanceof IndicesRequest) {
       final IndicesRequest req = (IndicesRequest) request;
       if (req.indices() != null) {
-        span.setTag("elasticsearch.request.indices", Joiner.on(",").join(req.indices()));
+        Tags.DB_INSTANCE.set(span, Joiner.on(",").join(req.indices()));
```

Comment:
I'm not sure this one is appropriate, especially when people use time-based or date-based indices. Elasticsearch typically will have a cluster name that is more appropriate to use for the `db.instance` tag.